packages/Algernon.package/ALGResultList.class/instance/add.target.selector.argumentList..st

```diff
@@ -6,5 +6,7 @@ add: anItem target: target selector: aSymbol argumentList: argList
 				target: target
 				selector: aSymbol
 				argumentList: argList
-				in: self.
+				in: self
+				index: self currentItemIndex.
+	self currentItemIndex: self currentItemIndex + 1.
```

Comment:
Tbh I don't really like the idea of having an instance variable that is only in use during initialization, but of course still there as soon as everything is finished. Especially since you've written public accessors for it.

I'd rather:
- Make it a private variable with no public accessors or
- Better: Rewrite it to use a local counter variable (in `updateWith:`), possibly removing the indirection via the `addAll:`

Reply:
Oof... I think this is pretty difficult to realize. If you think you can do it better, just push a commit.